<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportFiles</class>
 <widget class="QWidget" name="ExportFiles">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExportFiles</string>
  </property>
  <widget class="QLabel" name="labelEFMain">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Export Files</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>441</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Histograms</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_expSpec">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>SAve all experimental spectra i.e. 6300, 6310, 6320</string>
    </property>
    <property name="text">
     <string>EXperimental Spectra (6300,6310,6320,...)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_RespFuncRaw">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>421</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Spectra between 1-1000 are for 6300 spectrum, 1001-2000 for 6310">6300-&gt; 1- 1000; 6310-&gt;2001-3000; etc.</string>
    </property>
    <property name="text">
     <string>REsponse Functions (not fitted) 1-1000; 1001-2000; ... -6000</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Contaminations">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Contaminations 6301-6309,6311-6319,..</string>
    </property>
    <property name="text">
     <string>COntaminations (6301-6309; 6311-6319; ...)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_SimSpec">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SImulated SUM spectrum (7300, 7310,...)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_RecSpec">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>REconstrucetd Spectrum (8300, 8310,...)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_RespFunFitted">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fitted REsponse Functions</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_ExportHIS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Export HIS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_ExportDAT">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Export DAT</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DATContaminations">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DATexpSpec">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DATSimSpec">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>86</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DATRecSpec">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DATRespFunFitted">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DATRespFuncRaw">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>120</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_HIS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>18</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>HIS format</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_DAT">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>18</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>DAT format</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>441</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>ASCII</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_ExportDecayXML">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Decay XML files</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_ExportASCII">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Export ASCII</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Feeding">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Feeding and BGT(Gnuplot script)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_GenDecInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>General Decay Info</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_ExportDecayENS">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Decay ENS file</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_GammaEvo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gamma Evolution</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="buttonClose">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonExpAll">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>ExportAll</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
